pyproject.toml

```
@@ -44,7 +44,8 @@ dynamic = ["version"]
 all = [
     "contextily",
     "geopandas",
-    "ipython"
+    "ipython",
+    "rioxarray"
```

Always add a trailing comma following PEP8 (https://peps.python.org/pep-0008/#when-to-use-trailing-commas), although it's not a Python file.
```suggestion
    "rioxarray",
```